### PR TITLE
Feature 836 rtd release doc

### LINKED ISSUE #836: ReadTheDocs: Set up gh-pages redirection and update Release Guide
## Describe the Task ##
Once ReadTheDocs is set up for all METplus components, set up gh-pages redirections, add documentation about ReadTheDocs, and remove or modify the "Update the Documentation on the Web" instructions in the Release Guide.

### Time Estimate ###
*Estimate the amount of work required here.*
*Issues should represent approximately 1 to 3 days of work.*
< 1 day of work

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
ASAP

### Funding Source ###
Split

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

Updating link references from dtcenter.github.io to metplus.readthedocs.io are also included in this pull request.

- [x] Describe testing already performed for these changes:</br>

Reviewed the modified sections:

1. https://metplus.readthedocs.io/en/feature_836_rtd_release_doc/Release_Guide/metplus_official.html#update-the-documentation-on-the-web (This same text exists in the official release sections for METcalcpy, METplotpy, and MET - other components do not yet have documentation set up; There is no documentation section needed for bugfix or development releases)
2. https://metplus.readthedocs.io/en/feature_836_rtd_release_doc/Contributors_Guide/documentation.html
3. https://metplus.readthedocs.io/en/feature_836_rtd_release_doc/Contributors_Guide/add_use_case.html#accessing-the-documentation

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review the files changed and point out anything that may be unclear or needs additional documentation.  Review the web links above to ensure they look ok to you.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**. I do not see a warnings file here: https://github.com/dtcenter/METplus/actions/runs/723531239

- [x] Do these changes include sufficient testing updates? **[Yes]** None needed

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [x] After submitting the PR, select **Linked Issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Review by georgemccabe — CHANGES_REQUESTED** [review]
These changes mostly look good. It is nice to be able to delete a lot of the files for steps that are no longer needed!

I did notice that there is out-of-date info (paths to use case documentation files and location of .py doc files) in the Documentation page of the Contributor's Guide that were not from these changes. The info is duplicated in the Add Use Case chapter where the info has been updated (where to add use case documentation files, list of model applications, etc). We should figure out where this info should go and reference it from the other page so we don't run into this issue again.

Other than that, the content looks good!

**Comment by jprestop** [comment]
I added to information to https://metplus.readthedocs.io/en/feature_836_rtd_release_doc/Contributors_Guide/documentation.html#adding-new-documentation, which appeared to be an overview.  I noticed that it listed the User’s Guide and Contributor’s Guide, but not the Release Guide or the Verification Datasets Guide, so I added information about those.  I did not review these sections for duplicate information. It appeared that this section was out of date since it didn’t include information about the Release Guide or Verification Datasets Guide.  Maybe a separate issue should be created to check for duplicate information in these sections?

**Comment by jprestop** [comment]
I went ahead and cleaned it up now.  Once I started to make some corrections, I realize it would be easier to just consolidate the info now.

File: docs/Contributors_Guide/add_use_case.rst [hunk 1]
```diff
@@ -53,8 +66,19 @@ Use Case Content
...
 ^^^^^^^^^^^^^^^^^^^^^^
 
-In the category sub-directory (parm/use_cases/model_applications/<CATEGORY>),
-each use case should have the following:
+If creating a new MET tool wrapper use case, in the MET tool name
+sub-directory (parm/use_cases/met_tool_wrapper/<MET TOOL NAME>), each
+use case should have the following:
+
+* A METplus configuration file where the MET tool name follows PascalCase,
+  e.g. GridStat.conf or ASCII2NC.conf.  This file is a hybrid RST and Python
```
Comment (georgemccabe) [inline docs/Contributors_Guide/add_use_case.rst]:
```suggestion
  e.g. GridStat.conf or ASCII2NC.conf.
```

File: docs/Contributors_Guide/add_use_case.rst [hunk 2]
```diff
@@ -53,8 +66,19 @@ Use Case Content
...
+  file.  If the use case uses a Python embedding script, it should be
```
Comment (georgemccabe) [inline docs/Contributors_Guide/add_use_case.rst]:
```suggestion
  If the use case uses a Python embedding script, it should be
```

File: docs/Contributors_Guide/add_use_case.rst [hunk 3]
```diff
@@ -53,8 +66,19 @@ Use Case Content
...
+  indicasted in the by adding "_python_embedding" to the MET tool name.
+  e.g. GridStat_python_embedding.conf
+
+If createing a new model applications use case, in the category sub-directory
```
Comment (georgemccabe) [inline docs/Contributors_Guide/add_use_case.rst]:
```suggestion
If creating a new model applications use case, in the category sub-directory
```

**Review by georgemccabe — APPROVED** [review]
The updates look good to me. Thanks for making them! I approve.

**Comment by jprestop** [comment]
Thanks for checking and for noticing the duplication.
